<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailWidget</class>
 <widget class="QWidget" name="detailWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
border-image: url(:/images/images/passBg2.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_title">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>550</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame#frame_title
{
	border-top-left-radius:3px;
	border-top-right-radius:3px;
	background-color:qlineargradient(x1:0,y1:0,x2:1,y2:0,stop:0 rgba(0,83,207,0.9),stop:1 rgba(66,183,255,0.9));
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_title">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_left">
           <property name="minimumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>213</width>
             <height>40</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_left">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_logo">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>75</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/images/images/logo1.png);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_title">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>8</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>办公终端人脸识别</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>58</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="frame_right">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/images/images/topHang.png);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_right">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_return">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="btn_return">
                 <property name="minimumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton#btn_return
{
	border-image: url(:/images/images/return2.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_min">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_min
{
	border-image: url(:/images/images/min.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_close">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#btn_close
{
	border-image: url(:/images/images/close.png);
}
QPushButton:hover{background-color:rgba(255,255,255,0.5);}
QPushButton:pressed{background-color:rgba(100,100,100,1);}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame#frame {
	border: 1px solid rgba(212, 229, 248, 1);
	background-color: rgb(255, 255, 255);
    border-radius: 4px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="minimumSize">
               <size>
                <width>518</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>518</width>
                <height>120</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_face">
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>130</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_4">
                 <property name="minimumSize">
                  <size>
                   <width>386</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>386</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0">
                  <property name="leftMargin">
                   <number>20</number>
                  </property>
                  <property name="rightMargin">
                   <number>20</number>
                  </property>
                  <property name="horizontalSpacing">
                   <number>23</number>
                  </property>
                  <property name="verticalSpacing">
                   <number>6</number>
                  </property>
                  <item row="1" column="1">
                   <widget class="QPushButton" name="btn_delete">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton#btn_delete{
	color: rgb(75, 131, 255);
	background-color: rgb(233, 240, 255);
	border: 1px solid rgb(28, 95, 245);
	border-radius: 4px;
}

QPushButton#btn_delete:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}

QPushButton#btn_delete:pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}</string>
                    </property>
                    <property name="text">
                     <string>删除用户</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QPushButton" name="btn_add">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton#btn_add{
	color: rgb(75, 131, 255);
	background-color: rgb(233, 240, 255);
	border: 1px solid rgb(28, 95, 245);
	border-radius: 4px;
}

QPushButton#btn_add:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}

QPushButton#btn_add:pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}</string>
                    </property>
                    <property name="text">
                     <string>添加时间段</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QPushButton" name="btn_change">
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>微软雅黑</family>
                      <pointsize>8</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton#btn_change{
	color: rgb(75, 131, 255);
	background-color: rgb(233, 240, 255);
	border: 1px solid rgb(28, 95, 245);
	border-radius: 4px;
}

QPushButton#btn_change:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}

QPushButton#btn_change:pressed{
	color: rgb(255, 255, 255);
	background-color: rgb(28, 95, 245);
}</string>
                    </property>
                    <property name="text">
                     <string>变更信息</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="3">
                   <widget class="QLabel" name="label_name">
                    <property name="minimumSize">
                     <size>
                      <width>346</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>346</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>黑体</family>
                     </font>
                    </property>
                    <property name="text">
                     <string>姓名：</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="styleSheet">
               <string notr="true">QTableWidget#tableWidget{
  border: 1px solid rgba(212, 229, 248, 1);
  color: rgb(55, 76, 120);
  border-radius: 4px;
}

QHeaderView::section {  
  background-color: rgba(224, 240, 255, 1);  
  color: rgba(55, 76, 120, 1);
  /*font-family: '微软雅黑';*/
  border:0px;
} 

QTableWidget::item {
  border:0px;
}

QTableWidget::item:nth-child(even) {
  background-color: rgba(224, 240, 255, 1);
}

QTableWidget::item:nth-child(odd) {
  background-color: white; 
}

QTableWidget::item:selected {
  background-color: rgba(55, 76, 120, 0.2);
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
